app/src/test/java/org/mozilla/focus/home/pocket/PocketEndpointTest.kt

```diff
@@ -0,0 +1,90 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.home.pocket
+
+import org.json.JSONObject
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotNull
+import org.junit.Assert.assertNull
+import org.junit.Assert.fail
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mozilla.focus.TestResource
+import org.robolectric.RobolectricTestRunner
+
+private const val KEY_INNER = "list"
+
+@RunWith(RobolectricTestRunner::class)
+class PocketEndpointTest {
+
+    @Test
+    fun `convert Videos JSON successfully to internal objects`() {
+        val expectedSubset = listOf(
+                PocketVideo(
```

Comment:
Do we need to change this list every time the pocket video recommendations change? Or are we keeping these recommendations constant? I guess my question is are these hard-set recommendations? haha

Reply:
@bkmunar What do you think? What portion of the code is this test testing and how would that relate to the test lists that we provide?